scripts/deploy_cf.sh

```diff
@@ -37,10 +37,10 @@ cf_compile_manifest() {
   cd ~
 
   # Output the director uuid to be populated by spiff
-  echo -e "---\ndirector_uuid: $($BOSH_CLI status --uuid)" > templates/stubs/director-uuid.yml
+  echo -e "---\ndirector_uuid: $($BOSH_CLI status --uuid)" > templates/director-uuid.yml
 
   # Generate the manifest
-  CF_RELEASE_PATH=~/cf-release/ ./generate_deployment_manifest.sh $TARGET_PLATFORM > ~/cf-manifest.yml
+  ./scripts/generate_cf_manifest.sh > ~/cf-manifest.yml
```

Comment:
Why do we need this new script instead of using the existing `cf-manifest/build_manifest.sh` script?